PERFORMANCE.md

```diff
@@ -0,0 +1,350 @@
+# Performance Results
+
+We have collected SmartSim performance results on Horizon, a Cray XC50 supercomputer.
+
+Horizon Node Hardware Summary:
+
+| Nodes | Cores | Threads | Processor | Memory | GPU |
+| :--- | --- | --- | --- | --- | --- |
+| 34 | 18 | 36 | Xeon E5-2699 v4 @ 2.20GHz BDW | 64 GB DDR4-2400 | --- |
+| 16 | 18 | 36 | Xeon E5-2699 v4 @ 2.20GHz BDW | 64 GB DDR4-2400 | 1 Nvidia Tesla_P100-PCIE-16GB |
+| 100 | 48 | 96 | Xeon 8160 CPU @ 2.10GHz SKL | 192 GB DDR4-2666 | --- |
+| 60 | 56 | 112 | Xeon 8176 CPU @ 2.10GHz SKL | 192 GB DDR4-2666 | --- |
+| 48 | 48 | 96 | Xeon 8260 CPU @ 2.40GHz CSL | 192 GB DDR4-2666 | --- |
+| 53 | 48 | 96 | Xeon 8260 CPU @ 2.40GHz CSL | 384 GB DDR4-2933 | --- |
+| 28 | 64 | 256 | ThunderX2 CN9980 v2.2 @ 2.50GHz | 256 GB DDR4-2666 | --- |
+
+Note that the first iteration can take longer (up to several seconds) than the rest of the execution. This
+is due to the DB loading libraries when the first RedisAI call is made. In the following plots, we excluded
+the first iteration time.
```

Comment:
This paragraph seems out of place. You should talk about what the plots you've collected are before going into details of collection methodology

Reply:
This still needs to be resolved